install_config/uninstall-olm-using-ansible.adoc

```diff
@@ -0,0 +1,14 @@
+// This assembly is included in the following assemblies:
+//
+// * n/a
+
+:_content-type: ASSEMBLY
+[id="uninstall-olm-using-ansible"]
+= Uninstalling Operator Lifecycle Manager using Ansible
+:context: uninstall-olm-using-ansible
+
+toc::[]
+
+After installing your cluster, you can uninstall the Technology Preview Operator Framework by using he {product-title} `openshift-ansible` installer.
```

Comment:
Forgive me for not knowing about this area. But I'm confused about why the description and the title are different. 

You have 
`= Uninstalling Operator Lifecycle Manager using Ansible`
followed by
`...you can uninstall the Technology Preview Operator Framework...`

P.S. After reading around the docs, I see why this makes sense. If you have a handy way to fix it, great, but otherwise it's perhaps not worth holding things up over.

Reply:
Addressed